data/ppdb-ap-pipe-schema-extra.yaml

```diff
@@ -13,6 +13,10 @@ columns:
   nullable: true
   description: Time when validity of this diaObject ends.
   default: null
+- name: nDiaSources
+  type: INT
+  nullable: false
+  description: Total number of DiaSources associated with this DiaObject.
```

Comment:
The one being used in ap_pipe is from ap_association. The reason the duplication exists is because the ap_association one as extra columns for the observed DiaSource filter.  It also doesn't have the DiaObjectLast table as we don't currently use the Ppdb concept of "dailyJob".